honeybadger.go

```diff
@@ -24,8 +47,37 @@ func (hb *HoneyBadger) sendBatch() {
 }
 
 // Create batch polling random transaction in queue
-func (hb *HoneyBadger) createBatch() {
-	panic("implement me w/ test case :-)")
+func (hb *HoneyBadger) createBatch() []Transaction {
+	minVal := min(hb.b, hb.que.Len())
+	buf := make([]Transaction, minVal)
+	pol := make([]Transaction, int(hb.b/hb.n))
+
+	for i := 0; i < minVal; i++ {
+		buf[i], _ = hb.que.Poll()
+	}
+
+	for i := 0; i < int(hb.b/hb.n); i++ {
+		r := rand.New(rand.NewSource(time.Now().UnixNano()))
+		idx := r.Intn(minVal)
```

Comment:
How about using `len(buf)` instead of `minVal`?

Then, it would be more clear. Also, `minVal--`  in line 65 would be not necessary.